app/services/post_alerter.rb

```diff
@@ -576,28 +576,10 @@ def notify_pm_users(post, reply_to_user, notified)
 
     # users who interacted with the post by _directly_ emailing the group
     if group = group_notifying_via_smtp(post)
-      group_email_regex = group.email_username_regex
-      email_addresses = Set[group.email_username]
-
-      post.topic.incoming_email.each do |incoming_email|
-        to_addresses = incoming_email.to_addresses&.split(';')
-        cc_addresses = incoming_email.cc_addresses&.split(';')
-
-        next if to_addresses&.none? { |address| address =~ group_email_regex } &&
-                cc_addresses&.none? { |address| address =~ group_email_regex }
-
-        email_addresses.add(incoming_email.from_address)
-        email_addresses.merge(to_addresses) if to_addresses.present?
-        email_addresses.merge(cc_addresses) if cc_addresses.present?
-      end
-
-      email_addresses.subtract([nil, ''])
+      email_addresses = post.topic.incoming_email_addresses(group: group)
 
-      if email_addresses.size > 1
-        Jobs.enqueue(:group_smtp_email,
-          group_id: group.id,
-          post_id: post.id,
-          email: email_addresses.to_a - [group.email_username])
+      if email_addresses.any?
```

Comment:
This used to need to be > 1...but then it subtracted the group email username from the array straight after, leaving only 1. This group username removal is handled by `post.topic.incoming_email_addresses` now.